network.go

```diff
@@ -284,12 +305,26 @@ func (n *network) CopyTo(o datastore.KVObject) error {
 	dstN.dbExists = n.dbExists
 	dstN.drvOnce = n.drvOnce
 
+	for _, v4conf := range n.ipamV4Config {
+		dstV4Conf := &IpamConf{}
+		v4conf.CopyTo(dstV4Conf)
+		dstN.ipamV4Config = append(dstN.ipamV4Config, dstV4Conf)
+	}
+
 	for _, v4info := range n.ipamV4Info {
 		dstV4Info := &IpamInfo{}
 		v4info.CopyTo(dstV4Info)
 		dstN.ipamV4Info = append(dstN.ipamV4Info, dstV4Info)
 	}
 
+	if n.ipamV6Config != nil {
```

Comment:
I get now what you are saying. Yeah, we do not need it here and for v6 info as well, this is in the copy function.
I got confused with what we do in the marshalling/unmarshalling functions.

Thanks, I will change it for v6 conf and info